server/models/Document.ts

```diff
@@ -58,10 +59,13 @@ type SearchOptions = {
   limit?: number;
   offset?: number;
   collectionId?: string;
+  share?: Share;
   dateFilter?: DateFilter;
   collaboratorIds?: string[];
   includeArchived?: boolean;
   includeDrafts?: boolean;
+  minWords?: number;
```

Comment:
Would consider renaming at this level, it's too ambiguous – eg.

```suggestion
  snippetMinWords?: number;
```